### PR TITLE
ceph ansible code changes to run task on all machine types.

### PR DESCRIPTION
**Pull request by sunilkumarn417**
ceph ansible task code changes for below,
- execute tasks on all machine types(except `argo`).
- supports non-collocated scenario. 
- tested in local(using `bruuni, depressa, extensa` machine type)

Signed-off-by: sunilkumarn417 <sunnagar@redhat.com>

### CODE REVIEW

File: qa/tasks/rhceph_ansible.py [hunk 1]
```diff
@@ -1,3 +1,7 @@
+"""
+RH Ceph Ansible Module
```
Comment (rakeshgm) [inline qa/tasks/rhceph_ansible.py]: I would suggest using the word `task` instead of `module`

Comment (sunilkumarn417) [reply qa/tasks/rhceph_ansible.py]: updated.

**Review by rakeshgm — APPROVED** [review]

**Comment by rakeshgm** [comment]
LGTM

**Comment by sunilkumarn417** [comment]
tested locally on most of the octo-lab machine types.

**Pull request merged**